### PR TITLE
Update/overhaul of  these 2 examples, including switching to use GKE

### PR DESCRIPTION
**Pull request by amygdala**
Update/overhaul of  these 2 examples, including switching to use GKE.
Smaller changes as well, including pointer to cloud shell, using standard sql in the bigquery query examples, ..

### CODE REVIEW

File: pubsub/README.md [hunk 1]
```diff
@@ -49,93 +48,96 @@ delivery](https://cloud.google.com/pubsub/overview#benefits).  This means that
...
 
-    $ gcloud config set project <project_id>
+We'll stay in this directory for the rest of the tutorial.
 
 
 ### Create a BigQuery table
 
 Next, create a BigQuery table to store your tweets. BigQuery groups
-tables into abstraction layers called datasets, so first create a dataset as necessary.  You can do this from the web UI, or from the command line like this:
+tables into abstraction layers called datasets, so first create a dataset as necessary.  You can do this from the [web UI](xxx), or from the command line like this:
```
Comment (tswast) [inline pubsub/README.md]: I don't know if we can link directly to the create a table / dataset dialog. There are screenshots at https://cloud.google.com/bigquery/docs/tables#create-table on how to create a table with the web UI, though.

File: pubsub/README.md [hunk 2]
```diff
@@ -49,93 +48,96 @@ delivery](https://cloud.google.com/pubsub/overview#benefits).  This means that
...
+a schema. This example includes a predefined schema in the `kubernetes-bigquery-python/bigquery-setup` subdirectory,
+`schema.json`, that you can use to define your table.  From the `pubsub` directory, run:
 
-    $ bq mk -t <your-dataset-name>.tweets <example_root>/bigquery-setup/schema.json
+```sh
+bq mk -t <your-dataset-name>.tweets ../bigquery-setup/schema.json
+```
 
-(If you prefer, you can also create your table via the web UI, pasting in the schema from `schema.json`).
+(If you prefer, you can also create your table via the web Bigquery UI, pasting in the schema from `schema.json`).
```
Comment (tswast) [inline pubsub/README.md]: Nit: BigQuery not Bigquery

File: pubsub/README.md (hunk 1 above)
Comment (amygdala) [reply pubsub/README.md]: oops, I'd missed that dead link too-- thanks.

**Comment by tswast** [comment]
Thanks Amy!

**Comment by jeffmendoza** [comment]
:tada: Awesome, just seeing this now.

**Pull request merged**